examples/example-gcp-csm-observability/src/main/java/io/grpc/examples/csmobservability/CsmObservabilityClient.java

```diff
@@ -72,14 +75,22 @@ public static void main(String[] args) throws Exception {
     // The port on which prometheus metrics will be exposed.
     int prometheusPort = 9464;
     AtomicBoolean sendRpcs = new AtomicBoolean(true);
+    ChannelCredentials credentials = InsecureChannelCredentials.create();
     if (args.length > 0) {
       if ("--help".equals(args[0])) {
-        System.err.println("Usage: [name [target [prometheusPort]]]");
+        System.err.println("Usage: [--xds-creds [name [target [prometheusPort]]]]");
```

Comment:
Since xdscreds is separately optional, it should be `[--xds-creds] [name [target...`

Reply:
Done.